ui/StatusQ/src/typesregistration.cpp

```diff
@@ -97,6 +99,18 @@ void registerStatusQTypes() {
                                                   return new OnboardingEnums;
                                               });
 
+    qmlRegisterSingletonType<StatusColors>("StatusQ.Core.Theme", 0, 1, "StatusColors",
```

Comment:
Why is the version here `0.1` and above in ThemePalette it's `1.0`, any particular reason? :)